store/tikv/coprocessor.go

```diff
@@ -628,6 +628,57 @@ func (worker *copIteratorWorker) handleTaskOnce(bo *Backoffer, task *copTask, ch
 	return worker.handleCopResponse(bo, resp.Cop, task, ch, nil)
 }
 
+const (
+	minLogBackoffTime       = 100
+	minLogKVProcessTime     = 100
+	minLogKVWaitTime        = 200
+	minLogScanProcessedInfo = 100000
+	minLogScanTotalInfo     = 200000
+)
+
+func (worker *copIteratorWorker) logTimeCopTask(costTime time.Duration, task *copTask, bo *Backoffer, resp *tikvrpc.Response) {
+	logStr := fmt.Sprintf("[TIME_COP_TASK] cop_resp_time:%s, cop_start_ts:%d, cop_region_id:%d, cop_store_addr:%s", costTime, worker.req.StartTs, task.region.id, task.storeAddr)
+	if bo.totalSleep > minLogBackoffTime {
+		logStr += fmt.Sprintf(", cop_backoff_ms:%d", bo.totalSleep)
+	}
+	var detail *kvrpcpb.ExecDetails
+	if task.cmdType == tikvrpc.CmdCopStream {
+		detail = resp.CopStream.ExecDetails
+	} else {
+		detail = resp.Cop.ExecDetails
+	}
+	if detail != nil {
+		if detail.HandleTime != nil {
+			processMs := detail.HandleTime.ProcessMs
+			waitMs := detail.HandleTime.WaitMs
+			if processMs > minLogKVProcessTime {
```

Comment:
@AndreMouche Does coprocessor also has a threshold to fill `HandleTime.ProcessMs` and `HandleTime.WaitMs` for slow cop requests ?

Reply:
YES

Reply:
@coocood Maybe we just need to print `HandleTime.ProcessMs` and `HandleTime.WaitMs` once they are set, no need to use another thresholds like `minLogKVProcessTime` and `minLogKVWaitTime`

Reply:
After we set the `HandleTime` and `ScanDetial` to `true`, every request will return them.

Reply:
When will we set `ScanDetail` to `true` ? @coocood

Reply:
ScanDetail is not set to true now. it's set only when the duration exceeds the threshold(1s) in TiKV.

Reply:
I still think we should remove these `minXXXX` thresholds defined in this file and do not check them.

Reply:
TiDB should control what to log, without depending on TiKV's implementation.

Reply:
@zz-jason If we log everything, the most important ones will be obscure.